KomodoPlatform/komodo-wallet-desktop
# Expand privacy mode

Closes https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/2466

To test:
- Enter privacy mode, and check all screens and modals to confirm fiat / crypto values are redacted.
- Orderbooks should and input fields/buttons for values (e.g. MAX button) should be disabled in privacy mode.
- Exit privacy mode, and confirm everything looks like normal.
